orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteStageHandlerTest.kt

```diff
@@ -42,9 +43,10 @@ object CompleteStageHandlerTest : SubjectSpek<CompleteStageHandler>({
   val publisher: ApplicationEventPublisher = mock()
   val clock = fixedClock()
   val contextParameterProcessor: ContextParameterProcessor = mock()
+  val stageDefinitionBuilderFactory: StageDefinitionBuilderFactory = mock()
 
   subject(GROUP) {
-    CompleteStageHandler(queue, repository, publisher, clock, contextParameterProcessor)
+    CompleteStageHandler(queue, repository, publisher, clock, contextParameterProcessor, stageDefinitionBuilderFactory)
```

Comment:
If we're happy with the approach, I'll add an additional test in here to validate that `planAfterStages()` is invoked as expected.